requirements.txt

```diff
@@ -1,4 +1,4 @@
-cld2_cffi>=0.1.4
-spacy>=2.0.8
-textacy>=0.6.0
-beautifulsoup4>=4.6.0
+beautifulsoup4~=4.6.0
+cld2_cffi~=0.1.4
+spacy~=2.0.8
+textacy~=0.6.0
```

Comment:
Why are we changing this? Is it a better idea to use `~=` over `>=`? Or is it required for PyPI?

Reply:
`>=` prescribes that anything larger than the given number is fine while `~=` says that anything larger up until the next backwards incompatible change (major version bump) is fine.

Reply:
You're right thought it doesn't necessarily belong in this PR. I only touched it because I wanted to be sure PyPi supported this (it does).

Reply:
OK. Let's go for this, but let's decide if we want to drop the minor revision numbers on bs4, spacy and textacy.